ReleaseProcessAutomation.UnitTests/Steps/Continues/ContinueReleaseOnMasterStepTests.cs

```diff
@@ -100,12 +103,13 @@ public void CreateTagAndMerge_WithAlreadySetTag_ThrowsException ()
         .Returns(new SemanticVersion());
 
     var step = new ContinueReleaseOnMasterStep(
```

Comment:
All the indents in this file were correct but are now on x2 indent. You should check your settings it seems there is some setting missing.